File: 滚动轮播图(上下+左右)/滚动轮播图(上下+左右)/ViewController.m
//
//  ViewController.m
//  滚动轮播图(上下+左右)
//
//  Created by bxkj on 2017/11/1.
//  Copyright © 2017年 devil. All rights reserved.
//

#import "ViewController.h"
#import "VerticalCycleCollectionView.h"

#define kScreenWidth [UIScreen mainScreen].bounds.size.width
#define kScreenHeight [UIScreen mainScreen].bounds.size.height
@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    
    [super viewDidLoad];
    VerticalCycleCollectionView *collectionView1 = [[VerticalCycleCollectionView alloc]initWithFrame:CGRectMake(40, 60, kScreenWidth - 80, 0.3 * kScreenHeight)];
    collectionView1.backgroundColor = [UIColor whiteColor];
    [self.view addSubview:collectionView1];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: OC{}的一个坑/OC{}的一个坑/ViewController.swift
//
//  ViewController.swift
//  OC{}的一个坑
//
//  Created by 周冰烽 on 2017/12/8.
//  Copyright © 2017年 周冰烽. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let l = UILabel()
        //参数结束 { 尾随闭包 }
        view.addSubview(l);
        // Extra argument in call
        // 调用了额外的参数
        {
            let l = UILabel()
            view.addSubview(l)
        }
        
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

